lib/commons/aria/arialabelledby-text.js

```diff
@@ -46,6 +46,9 @@ function arialabelledbyText(vNode, context = {}) {
   }
 
   const refs = idrefs(vNode, 'aria-labelledby').filter(elm => elm);
+
```

Comment:
ahem

Reply:
That should have been removed in https://github.com/dequelabs/axe-core/pull/2679/commits/bfa341ad45cd5843876f4c5d9230807e27946fe0